--- gftp-2.0.18/docs/sample.gftp/bookmarks.bookmarks	2002-08-05 21:46:57.000000000 +0200
+++ gftp-2.0.18/docs/sample.gftp/bookmarks	2005-07-31 22:43:21.000000000 +0200
@@ -172,6 +172,116 @@
 password=@EMAIL@
 account=
 
+[Mandriva Sites/Community SRPMS (California)]
+hostname=mirrors.usc.edu
+port=21
+protocol=FTP
+remote directory=/pub/linux/distributions/mandrakelinux/devel/community/SRPMS/
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[Mandriva Sites/Devel iso (California)]
+hostname=mirrors.usc.edu
+port=21
+protocol=FTP
+remote directory=/pub/linux/distributions/mandrakelinux/devel/iso/
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[Mandriva Sites/Official iso (California)]
+hostname=mirrors.usc.edu
+port=21
+protocol=FTP
+remote directory=/pub/linux/distributions/mandrakelinux/official/iso/
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[Mandriva Sites/Mandriva Old (California)]
+hostname=mirrors.usc.edu
+port=21
+protocol=FTP
+remote directory=/pub/linux/distributions/mandrakelinux/old/
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[Mandriva Sites/Community SRPMS (France)]
+hostname=ftp.ciril.fr
+port=21
+protocol=FTP
+remote directory=/pub/linux/mandrakelinux/devel/community/SRPMS/
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[Mandriva Sites/Devel iso (France)]
+hostname=ftp.ciril.fr
+port=21
+protocol=FTP
+remote directory=/pub/linux/mandrakelinux/devel/iso/
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[Mandriva Sites/Official iso (France)]
+hostname=ftp.ciril.fr
+port=21
+protocol=FTP
+remote directory=/pub/linux/mandrakelinux/official/iso/
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[Mandriva Sites/Community SRPMS (Taiwan)]
+hostname=ftp.isu.edu.tw
+port=21
+protocol=FTP
+remote directory=/pub/Linux/Mandrakelinux/devel/community/SRPMS/
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[Mandriva Sites/Devel iso (Taiwan)]
+hostname=ftp.isu.edu.tw
+port=21
+protocol=FTP
+remote directory=/pub/Linux/Mandrakelinux/devel/iso/
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[Mandriva Sites/Official iso (Taiwan)]
+hostname=ftp.isu.edu.tw
+port=21
+protocol=FTP
+remote directory=/pub/Linux/Mandrakelinux/official/iso/
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
+[Mandriva Sites/Mandriva Old (Taiwan)]
+hostname=ftp.isu.edu.tw
+port=21
+protocol=FTP
+remote directory=/pub/Linux/Mandrakelinux/old/
+local directory=
+username=anonymous
+password=@EMAIL@
+account=
+
 [RedHat Sites/FalseHope RPMs]
 hostname=ftp.falsehope.com
 port=21
